script.js

```diff
@@ -0,0 +1,44 @@
+let saladMenuFileData = [], prices = [];
+let bill = 0, divContent="<table>";
+
+fetch("/salads.json")
+  .then((response) => response.json())
+  .then((saladMenuFileData) => createMenu(saladMenuFileData));
+
+  function setPrice(counter,saladMenuFileData){
+    if((counter + 1) % 3 == 0) return 15;
+    else if(saladMenuFileData[counter][0] == 'E' ||
+            saladMenuFileData[counter][0] == 'R' ||
+            saladMenuFileData[counter][0] == 'G' ) return 10;
+    else return 20;
+  }
+
+function createMenu(saladMenuFileData){

```

Comment:
jeżeli chodzi o definiowanie funkcji to dużo lepszym pomysłem jest jawne deklarowanie. Chodzi tutaj o to, że mamy dwie możliwości stworzenia funkcji. Tak jak Ty zrobiłeś
function <nazwa funkcji>(<argumenty>) { ... }
i
const/let/var <nazwa funkcji> = function(<argumenty) { ... }

W drugim przypadku jawnie deklarujesz jaki zakres życia ma mieć funkcja. Druga opcja jest zalecana z tego względu. Kolejna sprawa to syntax es6 jest już zaimplementowany we wszystkich przeglądarkach więc śmiało można używać "array functions". Będzie zatem
const createMenu = (saladMenuFileData) => { ... }